### ISSUE TITLE
Test Koreader

### ISSUE THREAD

**Issue by jahu00** [jahu00/RikaiKyun2Research#4]
It's possible Koreader can do everything I need and in that case writing a new app is not needed.

**Comment by jahu00** [comment]
So far I failed to run KOReader. Maybe my assumption that I will be able to run it on Windows 10 was wrong (I tried WSL and an emulator). I'll give it another shot on actual android device.

I've noticed that Jade Reader got an update in 2018, but the dictionary overlay still feels too clunky to use. I can't find the source code to this app anywhere, but I'm sure I made some tweaks in the past to fix some minor bug on BOOX Lynx.

Old Rikaikyun is still broken. There appears to be an error with calculating the text area and tapping on text is misaligned at best and completely broken at worst. With no easy way to debug this issue, I think I'll let the old project RIP.
